WTG.Analyzers.Utils/SymbolExtensions.cs

```diff
@@ -18,13 +18,6 @@ public static bool IsMatch(this IMethodSymbol methodSymbol, string fullTypeName,
 				&& methodSymbol.ContainingType.IsMatch(fullTypeName);
 		}
 
-		public static bool IsMatch(this IMethodSymbol methodSymbol, string assemblyName, string fullTypeName, string methodName)
```

Comment:
Are the internal analyzers using these overloads? I probably would have kept them in, but I suppose they are fairly simple and so keeping them doesn't really save a lot of effort.

Reply:
Internal analyzers?

Reply:
Dev analyzers.

Reply:
Ah. I'll check, but:

1) If they're matching system types like Exception then they shouldn't match assembly either.
2) If they're matching internal types then we should probably just move this code there.

Reply:
Internal analyzers are matching on:

1. `System.Exception` from `mscorlib`
2. `System.IDisposable` from `mscorlib`
3. Custom attributes from an internal assembly.

The first two should match type full name only.

The last one could do that as well but we could also just move the assembly matching code to the one function there that performs the check.